src/noun.substance

```diff
@@ -5763,6 +5794,7 @@ hyper: magnesium_carbonate
 hyper: potassium_carbonate
 hypo: salt
 
+
```

Comment:
why two spaces?

Reply:
Bites me. Probably some extra typing in Emacs. Thought I had already addressed this. Will remove it

Reply:
Removed